### DISCUSSION TITLE
Failed to use the "rustdesk-nightly" source code to build, prompting "ffigen is not a command"

### DISCUSSION THREAD

**Discussion by WeiYea** [rustdesk/rustdesk#2267] [General]
error: failed to run custom build command for `rustdesk v1.2.0 (C:\Users\admin\Desktop\rustdesk-nightly)`

Caused by:
  process didn't exit successfully: `C:\Users\admin\Desktop\rustdesk-nightly\target\release\build\rustdesk-aa5309a84d02512a\build-script-build` (exit code: 101)
  --- stdout
  cargo:rerun-if-changed=src/flutter_ffi.rs

  --- stderr
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ffigen is not a command, or not executable.', build.rs:100:56
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Comment by yash1597** [comment]
@WeiYea have you solved this error i have similar kind of error
Failed to use the "rustdesk-nightly" source code to build
error: failed to run custom build command for Rustdesk v1.2.0 (E:\Rustdesk\rustdesk)

Caused by:
  process didn't exit successfully: E:\Rustdesk\rustdesk\target\release\build\Rustdesk-d66a9b4f8d360184\build-script-build (exit code: 101)
  --- stdout
  cargo:rerun-if-changed=Cargo.toml
  cargo:rerun-if-changed=src/flutter_ffi.rs

  --- stderr
  thread 'main' panicked at 'called Result::unwrap() on an Err value: flutter is not a command, or not executable.', build.rs:122:43
  note: run with RUST_BACKTRACE=1 environment variable to display a backtrace

**Reply by grummbeer** [reply]
seems flutter not "installed" or PATH not set.

https://docs.flutter.dev/get-started/install/windows
https://docs.flutter.dev/get-started/install/windows#update-your-path

**Comment by grummbeer** [comment]
> ffigen

https://github.com/rustdesk/rustdesk/discussions/1431
